src/components/homeContent.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from 'react'
+import axios from "axios"
+import { BsFillPlayFill } from 'react-icons/bs'
+
+const HomeContent = () => {
+    const [category, setCategory] = useState([])
+    const getCategory = () => {
+        (async () => {
+            try {
+                const categoryData = await axios.get("/api/categories");
+                setCategory(categoryData.data.categories);
+            } catch (error) {
+                console.error(error.message);
+            }
+        })()
+    }
+
+    useEffect(() => {
+        getCategory()
+    }, []);
+
+    return (
+        <>
+            <main className="home-main-container">
+                <div className="main-top-section">
+                    <div className="main-top-info">
+                        <h3>Manchester United</h3>
+                        <h2>Greatest comebacks of all time. Passion, Emotions and Goals.</h2>
+                        <p>Watch the premier of the epic documentary.</p>
+                        <button>Watch Now</button>
+                    </div>
+                    <img src="assets/united.png" alt="" className="main-top-img" />
+                </div>
+                <div className="main-middle-section">
+                    <h2>Explore by categories</h2>
+                    <ul>
+                        <li>Category 1</li>
```

Comment:
Hey,
Why are you hard coding the categories instead of mapping them from the state which is set after the component is mounted?

Reply:
Yes, will do that. I had made a UI first so its not updated.